Samples/Extensions/AlternativeEducationProgram/AlternativeEducationProgramMetaEd/Association/StudentAlternativeEducationProgramAssociation.metaed

```diff
@@ -0,0 +1,8 @@
+Association StudentAlternativeEducationProgramAssociation based on EdFi.GeneralStudentProgramAssociation
```

Comment:
Please add a license header to the newly created files.
We don't seem to add license headers for the next extensions: `.xsd` `.xml` `.json`
But we do for: `.metaed` `.sql` ([here](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/fccb517097a44917a0f62fe1c0c106fd0389ae53/Samples/Extensions/StudentTranscript/StudentTranscriptMetaEd/Common/ClassRankingExtension.metaed#L1) and [here](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/fccb517097a44917a0f62fe1c0c106fd0389ae53/Samples/Extensions/StudentTranscript/StudentTranscriptMetaEd/MetaEdOutput/SampleStudentTranscript/Database/PostgreSQL/ODS/Structure/0010-EXTENSION-SampleStudentTranscript-Schemas.sql#L1), for example)

Reply:
Decided to address this in a separate PR to unblock another story.